src/tests/app.test.js

```diff
@@ -93,7 +93,7 @@ describe('Build Tests', () => {
     // expect(res.body).to.have.property('message', 'this account does not exist');
   });
   it('It should grant access to a user with a valid token ', async () => {
-    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFiY0BnbWFpbC5jb20iLCJpYXQiOjE2ODA0ODc5ODMsImV4cCI6MTY4MDUwMjM4M30.udpfqgyx72GI-d8pLi-ik2BszbAn-tTRLExlr0KBoaA';
+    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFiY0BnbWFpbC5jb20iLCJpYXQiOjE2ODA1NzQ5OTQsImV4cCI6MTY4MTE3OTc5NH0.843FO-wtu3P-rVAG5VIGFB0Ph3VT_dkCncMFrY0sxOI';
```

Comment:
Why not generate a token here? This will fail the tests in the future after the token expires